chapters/publication.tex

```diff
@@ -205,43 +201,44 @@ \subsection{Technical writing with \LaTeX}
-while making it easy and accessible to the entire writing team,
-we need to abstract away from the technical details where possible.
+Cloud-based implementations of \LaTex\, discussed in the next section,
+allow teams to take advantage of the features of \LaTeX,
+without requiring knowledge of the technical details.
 
 \subsection{Getting started with \LaTeX\ in the cloud}
 
 \LaTeX\ is a challenging tool to get started using,
 but the control it offers over the writing process is invaluable.
 In order to make it as easy as possible for your team
 to use \LaTeX\ without all members having to invest in new skills,
-we suggest using a web-based implementation as your first foray into \LaTeX\ writing.
+we suggest using a cloud-based implementation as your first foray into \LaTeX\ writing.
 Most such sites offer a subscription feature with useful extensions and various sharing permissions,
 and some offer free-to-use versions with basic tools that are sufficient
 for a broad variety of applications,
 up to and including writing a complete academic paper with coauthors.
 
-Cloud-based implementations of \LaTeX\ are suggested here for several reasons.
-Since they are completely hosted online,
-they avoids the inevitable troubleshooting of setting up a \LaTeX\ installation
-on various personal computers run by the different members of your team.
-They also typically maintain a single continuously synced master copy of the document
+Cloud-based implementations of \LaTeX\ have several advantageous features.
+First, since they are completely hosted online,
+they avoid the inevitable troubleshooting required to set up a \LaTeX\ installation
+on various personal computers run by the different members of a team.
+Second, they typically maintain a single, continuously synced, master copy of the document
 so that different writers do not create conflicted or out-of-sync copies,
 or need to deal with Git themselves to maintain that sync.
-They typically allow inviting collaborators to edit in a fashion similar to Google Docs,
+Third, they typically allow collaborators to edit in a fashion similar to Google Docs,
 though different services vary the number of collaborators and documents allowed at each tier.
-Most importantly, some tools provide a ``rich text'' editor
+Fourth, and most usefully, some implementations provide a ``rich text'' editor
 that behaves pretty similarly to familiar tools like Word,
 so that collaborators can write text directly into the document without worrying too much
 about the underlying \LaTeX\ coding.
 Cloud services also usually offer a convenient selection of templates
 so it is easy to start up a project and see results right away
 without needing to know a lot of the code that controls document formatting.
 
-On the downside, there is a small amount of up-front learning required,
+Cloud-based implementations of \LaTeX\ also have disadvantages.
+There is a small amount of up-front learning required,
```

Comment:
The up-front learning in terms  of syntax is the same as in a desktop app, unless you're using the rich-text editor

Reply:
addressed in 4aa5ec3186f697a9b1ed4dcb9bb037e9e7477c9f